packages/lib/response/cache.ts

```diff
@@ -0,0 +1,35 @@
+export const getResponsesCacheTag = (surveyId: string) => `surveys-${surveyId}-responses`;
+
+export const getResponseCacheTag = (responseId: string) => `responses-${responseId}`;
+
+export const getEnvironmentResponsesCacheTag = (environmentId: string): string =>
+  `environments-${environmentId}-responses`;
+
+export const getResponsesByPersonIdCacheTags = (personId: string): string => `person-${personId}-responses`;
+
+/*
+ISSUES
+1. Pagination - how do we deal with findMany that are cached where page can be undefined
+2. How do we make sure tags are not duplicated in multiple functions
+
+3. How do we avoid writing long function names.
+The below proposal addresses this:
+
+serviceName + CacheTag
+
+Then every object within it begins with `by` and the dependency
+*/
+export const responseCacheTag = {
```

Comment:
I think I would prefer `responseCache`.
Then the revalidate command will be:
`revalidateTag(responseCache.byPersonId)`
which is easy to read :-)

Reply:
Yeah agree, sounds good.